src/style.css

```diff
@@ -2,0 +1,72 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar__page {
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.search-bar__form {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.search-bar__form--big {
+  margin-top: 20px;
+}
+
+.search-bar__form--small {
+  margin-top: 20px;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you've added a margin-top to both `.search-bar__form--big` and `.search-bar__form--small`. Ensure that you're consistent throughout your design to avoid potential margin collapse issues.